src/System.ComponentModel.TypeConverter/src/MS/Internal/Xml/Linq/ComponentModel/XComponentModel.cs

```diff
@@ -0,0 +1,576 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I assume all of this is just ported effectively as-is from netfx?

Reply:
Yes, as is: I kept the dotnet-bot commit to illustrate that.  I'll run the codeformatter to fix up the style issues.  I only ran IDE formatting which missed naming/visibility modifiers.